{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/metrics.py

```diff
@@ -1,34 +1,34 @@
-import numpy as np
+import pandas as pd
 import lantern
 
 
 def train_metrics():
     return dict(
-        loss=lantern.ReduceMetric(lambda state, loss: loss.item()),
-        accuracy=lantern.ReduceMetric(
-            reduce_fn=lambda rolling_mean, examples, predictions: (
-                0.9 * rolling_mean
-                + 0.1
-                * np.mean(
-                    [
-                        example.class_name == prediction.class_name
-                        for example, prediction in zip(examples, predictions)
-                    ]
-                )
-            ),
-            initial_state=0,
+        loss=lantern.Metric().reduce(lambda state, loss: dict(loss=loss.item())),
+        standard=lantern.Metric().reduce(
+            lambda state, predictions, examples: dict(
+                accuracy=predictions.accuracy(examples).float().mean().item()
+            )
         ),
     )
 
 
 def evaluate_metrics():
     return dict(
-        loss=lantern.MapMetric(lambda loss: loss.item()),
-        accuracy=lantern.MapMetric(
-            map_fn=lambda examples, predictions: [
-                example.class_name == prediction.class_name
-                for example, prediction in zip(examples, predictions)
-            ],
-            compute_fn=lambda results: np.mean(np.concatenate(results)),
+        standard=(
```

Comment:
What do we think about the name standard? The downside is it's not very descriptive, but I struggle to come up with something that is. from_predictions_and_examples is a suggestion but it's a little long

Reply:
`pairs`?